frontend/src/scenes/surveys/surveysLogic.tsx

```diff
@@ -36,6 +36,13 @@ export const surveysLogic = kea<surveysLogicType>([
                 const updatedSurvey = await api.surveys.update(id, { ...updatePayload })
                 return values.surveys.map((survey) => (survey.id === id ? updatedSurvey : survey))
             },
+            loadResponsesCount: async () => {
```

Comment:
sorry if I wasn't clear in my earlier comment: This isn't ideal because it depends on a loader for another thing: the surveys itself.

So, for example, if ever the case happens where this loads before surveys are loaded (say, if it were cached), it leads to a problem because surveys loader overrides this.

Hence, don't add this as an action/listener for the surveys loader, but create a new loader, `surveyResponsesCount`, that maps survey_id to the count.

And then use this independent loader in the response count field for the table. This allows you to then nicely have loading states for response counts as well while they're loading in the table.